### PR TITLE
Formatting fixes and style convention adjustments

### LINKED ISSUE #524: Docs convergence suggestion box
This issue is a bucket to capture suggestions, ideas, and proposals that come up "as we work" in this repo, regarding content structure and things that should be addressed during the migration. 

Please include link issue # or PR # for easier tracking.

### PR DESCRIPTION
**Pull request by thatdocslady**
Fixes #524 

This PR includes a collection of style, format, and structure fixes that address the following requests in the above issue:

- Restructured the installation and usage guides to remove mega-deep-nested lists for easier skimming/chunking.
- Restructured the troubleshooting guide to include workaround for Windows machine per request in https://github.com/projectatomic/adb-atomic-developer-bundle/issues/500#issuecomment-242678929
- Adjusted code block formatting to match downstream style guide
- Added links to new content from the readme and link to contribution document
- Removed "the" before ADB where it is not needed
- Searched for refereneces to Fedora 22 (none found)

### CODE REVIEW

File: docs/troubleshooting.adoc [hunk 1]
```diff
@@ -8,9 +8,24 @@ Use this document to identify and resolve basic ADB issues that you encounter.
...
+== Installation and Update
+
+This section contains issues related to installing, configuring, and updating ADB.
+
+=== On a Windows machine, Vagrant fails to initialize after updating to the latest version
+
+In addition to the standard link:updating.adoc[steps to update ADB], you need to run the
+following command to delete the .vagrant.d directory:
++
+....
```
Comment (Preeticp) [inline docs/troubleshooting.adoc]: The extra plus needs to be removed, it is showing in the rendered version.

**Comment by praveenkumar** [comment]
LGTM :+1: @Preeticp please have a look and let me know if that ready to push.

File: docs/using.adoc [hunk 2]
```diff
@@ -100,11 +102,13 @@ the version of the image to be used.
 .  Start ADB using `vagrant up`. You will find detailed installation
 instructions in the link:docs/installing.adoc[Installation document].
 
-.  Configure the environment and download the required TLS certificates
+=== Configure your environment
```
Comment (Preeticp) [inline docs/using.adoc:105]: +1, chunking the 3 sections below makes them quickly accessible to the user through the TOC itself.

**Comment by Preeticp** [comment]
LGTM from my side @praveenkumar

**Pull request merged**